tools/llvm2bpl/llvm2bpl.cpp

```diff
@@ -153,7 +152,8 @@ int main(int argc, char **argv) {
 
   pass_manager.add(llvm::createLowerSwitchPass());
   // pass_manager.add(llvm::createCFGSimplificationPass());
-  pass_manager.add(llvm::createInternalizePass());
+  // Shaobo: sea-dsa is inconsistent with the pass below.
+  //pass_manager.add(llvm::createInternalizePass());
```

Comment:
Do you maybe know why we added this pass? Does removing it mean that there'll be lots of external functions?

Reply:
No, I don't remember why we needed this pass. I don't know if removing it creates a lot of external functions. Let me investigate it.

Reply:
It appears this came from here:
https://github.com/smackers/smack/commit/3a3d81941cfc469502d745366b33f5f1a86c8fbc

Reply:
Did we get to the bottom of this?

Reply:
No, but it seems that removing this line does not cause any performance issues. I propose that we get back to this question when it manifests in an example.